### PR TITLE
October 2024 newsletter blog

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

October 2024 newsletter.

It's been really awkward to review the newsletter before it goes out, so I'm trying this new approach. Here at least it's reviewable. Getting it properly into form that can be used by mailchimp will be the next hurdle. Hopefully it will go OK.

Rendered version at https://20241001-oct-2024-newsletter.ddev-com-front-end.pages.dev/blog/ddev-oct-2024-newsletter/

### CODE REVIEW

**Comment by rfay** [comment]
@stasadev @rpkoller @bmartinez287 thanks for taking a look.

File: src/content/blog/ddev-oct-2024-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+categories:
+  - Community
+---
+
+
+**Happy October from DDEV**! I just got back from DrupalCon Barcelona and things are finally starting to cool down here in Palisade, Colorado, USA.
+
+**Podcast**! [Chad Carlson](https://www.linkedin.com/in/chadwcarlson/) of [DDEV Lead Sponsor](https://ddev.com/blog/platform-sh-becomes-a-lead-sponsor-of-ddev/) [Platform.sh](http://Platform.sh) had me on the Change Mode podcast and we had loads of fun. Learn more about the history of DDEV (and my TRS-80 Model 1 Level 2) on the [Change Mode Podcast](https://www.podcastics.com/podcast/episode/chmod-106-code-community-and-ddev-randy-fays-journey-311577/).
+
+**DrupalCon Barcelona**: I got to see lots of friends old and new in Barcelona. Four training sessions, a full-day contribution mentoring session, and loads of fun. [**Read about it on ddev.com](https://ddev.com/blog/drupalcon-barcelona-2024).**
```
Comment (rfay) [inline src/content/blog/ddev-oct-2024-newsletter.md:20]: This link won't work until I pull the other blog. It mostly needs a featured Image.

File: src/content/blog/ddev-oct-2024-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* October 23: **[Using Tmate to Debug GitHub Workflows**](https://www.meetup.com/ddev-events/events/303503674)**
+
+**GitHub Codespaces is working well with DDEV**: We used to add a lot of caveats about Codespaces, but it was fixed with new workaround and the [DDEV docs](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces) say how to do it. Give it a try!
+
+**DDEV Notes from around the web:**
+
+- **[Using DDEV in GitLab CI Tests](https://ddev.com/blog/ddev-in-gitlab-ci)** - Jochen Roth sorted out the difficulty and made this available to all. (Note that lots of people use DDEV for automated tests of their websites in GitHub Workflows using the [github-action-setup-ddev action](https://github.com/ddev/github-action-setup-ddev).)
+- **[Install Prestashop in DDEV](https://misterdigital.es/instalar-prestashop-en-ddev/)** (in Spanish, but Google Translate makes it very accessible)
+- **[CraftCMS Launchpad](https://craftcms-launchpad.mandrasch.eu/)** by prolific contributor [@mandrasch](https://mandrasch.dev/). Interactive CraftCMS demos in your browser, powered by DDEV.
```
Comment (rpkoller) [inline src/content/blog/ddev-oct-2024-newsletter.md:32]:
```suggestion
- **[CraftCMS Launchpad](https://craftcms-launchpad.mandrasch.eu/)** by the prolific contributor [@mandrasch](https://mandrasch.dev/). Interactive CraftCMS demos in your browser, powered by DDEV.
```
should a definite article be added?

File: src/content/blog/ddev-oct-2024-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+**DrupalCon Barcelona**: I got to see lots of friends old and new in Barcelona. Four training session BoFs, a full-day contribution mentoring session, and loads of fun. **[Read about it on ddev.com](https://ddev.com/blog/drupalcon-barcelona-2024).**
+
+**Join us for Contributor Training**
+* October 9: **[Supporting users using `ddev debug test` and other tools](https://www.meetup.com/ddev-events/events/303503564)**
+* October 23: **[Using Tmate to Debug GitHub Workflows**](https://www.meetup.com/ddev-events/events/303503674)**
+
+**GitHub Codespaces is working well with DDEV**: We used to add a lot of caveats about Codespaces, but it was fixed with new workaround and the [DDEV docs](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces) say how to do it. Give it a try!
```
[the remark this answers is not in the document]
Comment (rfay) [reply src/content/blog/ddev-oct-2024-newsletter.md:25]:
```suggestion
**GitHub Codespaces is working well with DDEV**: We used to add a lot of caveats about Codespaces, but the problems were fixed with a new workaround shown in the [DDEV docs](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces) say how to do it. Give it a try!
```

File: src/content/blog/ddev-oct-2024-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+**DDEV Notes from around the web:**
+
+- **[Using DDEV in GitLab CI Tests](https://ddev.com/blog/ddev-in-gitlab-ci)** - Jochen Roth sorted out the difficulty and made this available to all. (Note that lots of people use DDEV for automated tests of their websites in GitHub Workflows using the [github-action-setup-ddev action](https://github.com/ddev/github-action-setup-ddev).)
```
[the remark this answers is not in the document]
Comment (rfay) [reply src/content/blog/ddev-oct-2024-newsletter.md:29]: IMO "to all" is fine. "you all" is southern US vernacular, now widely accepted, but less formal.

File: src/content/blog/ddev-oct-2024-newsletter.md (hunk 2 above)
Comment (rfay) [reply src/content/blog/ddev-oct-2024-newsletter.md:31]: I like it the way it is, but thanks.

File: src/content/blog/ddev-oct-2024-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+**Join us for Contributor Training**
+* October 9: **[Supporting people using `ddev debug test` and other tools](https://www.meetup.com/ddev-events/events/303503564)**
+* October 23: **[Using Tmate to Debug GitHub Workflows**](https://www.meetup.com/ddev-events/events/303503674)**
+
+**GitHub Codespaces is working well with DDEV**: We used to add a lot of caveats about Codespaces, but the problems were fixed with a new workaround shown in the [DDEV docs](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces) say how to do it. Give it a try!
+
+**DDEV Notes from around the web:**
+
+- **[Using DDEV in GitLab CI Tests](https://ddev.com/blog/ddev-in-gitlab-ci)** - Jochen Roth sorted out the difficulty and made this available to all. (Note that lots of people use DDEV for automated tests of their websites in GitHub Workflows using the [github-action-setup-ddev action](https://github.com/ddev/github-action-setup-ddev).)
+- **[Install Prestashop in DDEV](https://misterdigital.es/instalar-prestashop-en-ddev/)** (in Spanish, but Google Translate makes it very accessible)
```
Comment (stasadev) [inline src/content/blog/ddev-oct-2024-newsletter.md:30]:
```suggestion
- **[Install Prestashop in DDEV](https://misterdigital.es/instalar-prestashop-en-ddev/)** (in Spanish, but Google Translate makes it very accessible).
```

File: src/content/blog/ddev-oct-2024-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+**THANKS to all of you who are supporting DDEV’s path to sustainability** and who have gotten your organizations to do so.
+
+Want to keep up as the month goes along? Follow our [blog](https://ddev.com/blog/), [LinkedIn](https://www.linkedin.com/company/ddev-foundation), [Mastodon](https://fosstodon.org/@ddev), [X,](https://x.com/randyfay) and join us on [Discord](https://discord.gg/5wjP76mBJD). And of course email us any time, you can reply to this email. If you didn’t get this newsletter in your own inbox, you can. Sign up at https://ddev.com/newsletter.
+
+Happy October from Randy Fay, DDEV Project Lead
```
Comment (stasadev) [inline src/content/blog/ddev-oct-2024-newsletter.md:46]:
```suggestion
Happy October from Randy Fay, DDEV Project Lead.
```

**Review by stasadev — APPROVED** [review]

**Review by bmartinez287 — APPROVED** [review]
Looks good

**Pull request merged**